test/parallel/test-buffer-sharedarraybuffer.js

```diff
@@ -0,0 +1,26 @@
+/*global SharedArrayBuffer*/
+/*eslint no-undef: "error"*/
```

Comment:
This is already set in the main project `.eslintrc` file so this comment should be unnecessary, unless there's something I'm missing.